src/templates/ControlsList.jsx

```diff
@@ -0,0 +1,214 @@
+import { Checkbox } from "@trussworks/react-uswds";
+import PropTypes from "prop-types";
+import { useState } from "react";
+import {
+  createSearchParams,
+  useLocation,
+  useNavigate,
+  useSearchParams,
+} from "react-router-dom";
+
+import { Link } from "react-router-dom";
+import { MAIN_ROUTES } from "../AppRoutes";
+import Pagination from "../molecules/Pagination";
+
+const TableRow = ({ control }) => {
+  if (control.control.title === undefined) {
+    return "";
+  }
+  return (
+    <>
+      <td>
+        <Link
+          to={`/projects/${control.project}/controls/${control.control.control_id}`}
+        >
+          {control.control.control_label}
+        </Link>
+      </td>
+      <td>{control.control.title}</td>
+      <td>
+        {control.status === "completed"
+          ? "Completed"
+          : control.status === "incomplete"
+          ? "Incomplete"
+          : "Not started"}
+      </td>
+    </>
+  );
+};
+
+const EmptyResults = ({ length }) => {
+  if (length !== 0) {
+    return "";
+  }
+  return (
+    <p>
+      There are no controls in your project yet. Add one by visiting the{" "}
```

Comment:
Hmm, this isn't how controls are added to a project at the moment. All controls associated with the project's catalog are added (components aren't taken into account)

Reply:
Good catch.

Reply:
Is this something that we need to deal with in the future?